src/main/java/org/dependencytrack/persistence/VulnerabilityQueryManager.java

```diff
@@ -72,6 +72,7 @@ final class VulnerabilityQueryManager extends QueryManager implements IQueryMana
      * @return a new vulnerability object
      */
     public Vulnerability createVulnerability(Vulnerability vulnerability, boolean commitIndex) {
+        vulnerability.setSeverity(vulnerability.getSeverity());
```

Comment:
Add a comment before this line, explaining why this is done, since its not obvious to a reader